src/System.Memory/tests/ReadOnlySpan/IndexOfAny.T.cs

```diff
@@ -963,5 +963,21 @@ public static void MakeSureNoChecksGoOutOfRangeIndexOfAny_ManyString()
                 Assert.Equal(-1, index);
             }
         }
+
+        [Theory]
+        [MemberData(nameof(TestHelpers.IndexOfAnyNullSequenceData), MemberType = typeof(TestHelpers))]
+        public static void IndexOfAnyNullSequence_String(string[] spanInput, string[] searchInput, int expected)
+        {
+            ReadOnlySpan<string> theStrings = spanInput;
+            Assert.Equal(expected, theStrings.IndexOfAny(searchInput));
+        }
+
+        [Theory]
+        [MemberData(nameof(TestHelpers.IndexOfAnyNullData), MemberType = typeof(TestHelpers))]
+        public static void IndexOfAnyNull_String(string[] spanInput, string[] searchInput, int expected)
+        {
+            ReadOnlySpan<string> theStrings = spanInput;
+            Assert.Equal(expected, theStrings.IndexOfAny(searchInput[0], searchInput[1]));
```

Comment:
Can you also add:
```C#
Assert.Equal(expected, theStrings.IndexOfAny((ReadOnlySpan<string>)searchInput));
```
?

Same for the other occurrences, where we can test both the overloads that accept multiple individual elements as well as a span of elements.